<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2345346</width>
    <height>346346</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget
{
	
	background-color: rgb(0, 85, 255);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>541</width>
        <height>405</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>541</width>
        <height>405</height>
       </size>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>541</width>
         <height>34</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 75 italic 11pt &quot;Bahnschrift&quot;;</string>
          </property>
          <property name="text">
           <string>咕咕音乐</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>22</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"> QPushButton{
	color: rgb(0, 0, 0);
	font: 22pt &quot;Arial&quot;;
	
border:none;
}
QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"> QPushButton{
	color: rgb(0, 0, 0);
	font: 87 22pt &quot;Arial&quot;;
	
border:none;
}
QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
</string>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>label_3</zorder>
       <zorder>pushButton_3</zorder>
       <zorder>horizontalSpacer</zorder>
       <zorder>pushButton_5</zorder>
       <zorder>label_5</zorder>
      </widget>
      <widget class="QPushButton" name="pushButton_login">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>340</y>
         <width>300</width>
         <height>45</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>45</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_login{
background-color: rgb(99, 188, 255);
color: rgb(255, 255, 255);
border-radius: 5px;
}
#pushButton_login:hover{
background-color: rgb(99, 170, 255);
color: rgb(255, 255, 255);
border-radius: 5px;

}
#pushButton_login:pressed{
background-color: rgb(99, 188, 255);
color: rgb(255, 255, 255);
border-radius: 5px;
padding-left:2px;
padding-top:2px;
}</string>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>220</y>
         <width>251</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>账号</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>220</y>
         <width>271</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    font-size: 15px;
    border: none; /* 其他边框样式设置为无 */
    padding: 5px;
    border-bottom: 2px solid rgb(186, 186, 186);
	background-color: rgb(255, 255, 255);
}

QComboBox::drop-down {
    border-style: none; /* 移除下拉箭头的边框 */
    width: 20px;
    background-color: rgb(255, 255, 255);
}

QComboBox::down-arrow {
    border-image: url(:/tu/jietou1.png);
}

QComboBox::down-arrow:pressed {
    border-image: url(:/tu/上拉箭头.png);
}

QComboBox:hover {
    border-bottom: 2px solid rgb(52, 152, 219); /* 鼠标悬停时的下边框颜色 */
}

QComboBox:pressed {
    border-bottom: 2px solid rgb(52, 152, 219); /* 点击时的下边框颜色 */
}

QComboBox QAbstractItemView {
    border: none; /* 移除下拉列表边框 */
    border-radius: 5px; /* 可选：圆角 */
    selection-background-color: rgb(52, 152, 219); /* 选择项背景颜色 */
    selection-color: white; /* 选择项文字颜色 */
}

QComboBox QAbstractItemView::item {
    min-height: 50px;
    min-width: 60px;
}
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>60</width>
         <height>15</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton{
	color: rgb(194, 194, 194);
border:none;
}
QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }


#pushButton_4:hover{
color: rgb(120, 120, 120);
}






</string>
       </property>
       <property name="text">
        <string>注册账号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>271</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>541</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>541</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>310</y>
         <width>356</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(176, 176, 176);</string>
          </property>
          <property name="text">
           <string>自动登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(176, 176, 176);</string>
          </property>
          <property name="text">
           <string>记住密码</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <zorder>label_2</zorder>
      <zorder>label_4</zorder>
      <zorder>label</zorder>
      <zorder>layoutWidget_3</zorder>
      <zorder>pushButton_login</zorder>
      <zorder>pushButton_4</zorder>
      <zorder>LineEdit_2</zorder>
      <zorder>layoutWidget</zorder>
      <zorder>comboBox</zorder>
      <zorder>LineEdit</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
